src/NextGen/DonationForm/Blocks/DonationFormBlock/Block.php

```diff
@@ -242,6 +236,30 @@ private function enqueueScripts(int $formId)
             'give'
         ))->loadInFooter()->enqueue();
 
+        // load template
+        /** @var FormTemplateRegistrar $formTemplateRegistrar */
+        $formTemplateRegistrar = give(FormTemplateRegistrar::class);
+
+        // silently fail if template is missing for some reason
+        if ($formTemplateRegistrar->hasTemplate($formTemplateId)) {
+            $template = $formTemplateRegistrar->getTemplate($formTemplateId);
+
+            if ($template->css()) {
+                wp_enqueue_style('givewp-form-template-' . $template->getId(), $template->css());
+            }
+
+            if ($template->js()) {
+                wp_enqueue_script(
```

Comment:
Is there a particular reason for using `wp_enqueue_script` here, as opposed to the `EnqueueScript` classed used elsewhere in this file?

Reply:
The `EnqueueScript` class requires defining urls like `$pluginDirPath` and `$pluginDirUrl` which would change depending on if the template is coming from a custom plugin.  The PaymentGateway class is enforcing using that but we want to keep the template simple for people by just specifying a path.